source/standards/optimise-frontend-perf.html.md.erb

```diff
@@ -62,10 +61,22 @@ You can automate frontend performance testing using third-party services such as
 - Sitespeed.io
 - Calibreapp
 
+## Using Speedcurve to automate frontend performance testing
+
+You can use [Speedcurve][] to test the performance of your service.  
+
+Speedcurve provides an extensive set of tools to test your service such as the following:
+
+- Test from different locations and different browsers.
+- Schedule regular tests using tools like Google Lighthouse and Webpage Test.
```

Comment:
Is it worth adding real user metrics / monitoring (RUM) to this list?

Reply:
Yes, I think this is a great idea. We can even link back to [our own blogpost](https://insidegovuk.blog.gov.uk/2021/06/16/how-real-user-monitoring-will-improve-gov-uk-for-everyone/) on the subject too.

Reply:
Since it's specifically about SpeedCurve. We should probably mention that their RUM tool is called LUX and stands for "Live User eXperience".

Reply:
Scrap the mention of LUX, they are now just calling it "Speedcurve RUM".

Reply:
Good idea - done